test/SettingsMigrator.UnitTests/oldTestSettings.testsettings

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TestSettings name="TestSettings1" id="2ac9d667-ce4a-4a9b-a1ca-4f82847322cc" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
+  <Description>These are default test settings for a local test run.</Description>
+  <Deployment>
+    <DeploymentItem filename=".\test.txt" />
+  </Deployment>
+  <Scripts setupScript=".\setup.bat" cleanupScript=".\cleanup.bat" />
+  <Execution parallelTestCount="2" hostProcessPlatform="MSIL">
```

Comment:
Why `hostProcessPlatform` not migrating?

Reply:
Will be done separately